RecoPixelVertexing/PixelTriplets/plugins/CAGraph.h

```diff
@@ -1,68 +1,55 @@
 #ifndef RECOPIXELVERTEXING_PIXELTRIPLETS_CAGRAPH_H_
 #define RECOPIXELVERTEXING_PIXELTRIPLETS_CAGRAPH_H_
 
-#include <vector>
 #include <array>
 #include <string>
+#include <vector>
 
-struct CALayer
-{
-	CALayer(const std::string& layerName, std::size_t numberOfHits )
-	: theName(layerName)
-	{
-		isOuterHitOfCell.resize(numberOfHits);
-	}
-
-	bool operator==(const std::string& otherString)
-	{
-		return otherString == theName;
-	}
+struct CALayer {
+  CALayer(const std::string &layerName, std::size_t numberOfHits)
+      : theName(layerName) {
+    isOuterHitOfCell.resize(numberOfHits);
+  }
 
-	std::string name() const
-	{
-		return theName;
-	}
+  bool operator==(const std::string &otherString) {
+    return otherString == theName;
+  }
 
-	std::vector<int> theOuterLayerPairs;
-	std::vector<int> theInnerLayerPairs;
+  std::string name() const { return theName; }
```

Comment:
This should return `const std::string&`.

Eventually it would be nice to avoid strings as the layer identifiers.